Zend/zend_alloc.h

```diff
@@ -27,12 +27,12 @@
 #include "zend.h"
 
 #ifndef ZEND_MM_ALIGNMENT
-# define ZEND_MM_ALIGNMENT ((size_t) 8)
+# define ZEND_MM_ALIGNMENT 8
```

Comment:
As C unfortunately doesn't have a suffix for `size_t`, I'd suggest replacing this with `Z_UL(8)`. It may differ from size_t, but only on x32 ABI.

Reply:
How about just getting the size of size_t during the configure stage and making it a define?